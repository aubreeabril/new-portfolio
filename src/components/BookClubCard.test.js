import React from "react";
import { shallow } from "enzyme";
import renderer from "react-test-renderer";
import BookClubCard from "./BookClubCard";

describe("Book Club Card", () => {
  it("renders the book club card", () => {
    const wrapper = shallow(<BookClubCard />);
    expect(wrapper.find(".book-club").length).toEqual(1);
  });

  it("matches its snapshot", () => {
    const section = renderer.create(<BookClubCard />).toJSON();
    expect(section).toMatchSnapshot();
  });
});
