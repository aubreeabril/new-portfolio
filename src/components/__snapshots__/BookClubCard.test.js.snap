// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Book Club Card matches its snapshot 1`] = `
<div
  className="project-card book-club card"
>
  <div
    className="card-image"
  >
    <iframe
      allow="autoplay; encrypted-media"
      allowfullscreen={true}
      frameborder="0"
      src="https://www.youtube.com/embed/bkCJCK8nF2Y"
      title="Book Club demo"
      width="100%"
    />
  </div>
  <h5
    className="card-title"
    style={
      Object {
        "padding": "10px",
      }
    }
  >
    Book Club
  </h5>
  <p
    className="project-text card-text"
  >
    A single page web application for book club management. Built with a Rails API backend and React/Redux frontend, styled with Ant Design.
  </p>
</div>
`;
